libs/application/templates/aosh/work-accident-notification/src/fields/EmployeeStartTimeError/index.tsx

```
@@ -0,0 +1,57 @@
+> = (props) => {
+  const { application, field, setBeforeSubmitCallback } = props
+  const { index } = field.props
+  const { getValues } = useFormContext<WorkAccidentNotification>()
+  const { formatMessage } = useLocale()
+  const [inputError, setInputError] = useState<boolean>(false)
+
+  setBeforeSubmitCallback?.(async () => {
+    const values = getValues('employee')
+    if (
+      values?.[index] &&
+      values[index].startOfWorkdayDate &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)